.github/workflows/testnet.yml

```diff
@@ -70,7 +70,7 @@ jobs:
           VITE_BUY_SFL_CONTRACT: "0x4196d162B9d4Bd3C31e64284A4E5f58c2cDb90FE"
           VITE_BUY_GEMS_CONTRACT: "0x0BF15a83889cAfcBAa19e5C44E319D96AbD0cD3E"
           VITE_DEPOSIT_CONTRACT: "0x05c63E157969C8a92e5E4591223705C9134F57b1"
-          VITE_BUD_DEPOSIT_CONTRACT: "0x000000000000000000000000000000000000dEaD"
+          VITE_BUD_DEPOSIT_CONTRACT: "0x05c63E157969C8a92e5E4591223705C9134F57b1"
```

Comment:
We're currently only using the `BUD_DEPOSIT_CONTRACT` in code. This situation was left over from the removal of a feature flag a long time ago. Will tidy this up in a separate PR. This change allows the deposit of buds on testnet which you weren't able to do prior.